Dockerfile

```diff
@@ -22,7 +24,7 @@ RUN apk -U upgrade && \
     bundler -v && \
     bundle config set no-cache 'true' && \
     bundle config set no-binstubs 'true' && \
-    bundle --retry=5 --jobs=4 --without=development && \
```

Comment:
I would this `--without=development` to stay ideally. This was written as a production dockerfile really, not a development one. Maybe we should create a Dockerfile.dev if you're keen on using it for dev?

Reply:
This dockerfile installs a lot of things that we never use in production - all the test dependencies are in it (and so dependencies in dev+test are in it too), because we run tests on it.

If we want to use separate dockerfiles, it's probably ok to add a tech debt card for it somewhere, that we can pick up when we do load or penetration testing and find out our dev/test dependencies are a bad idea.

Reply:
Hmm, actually, are we running tests on it?

Reply:
I guess we aren't. But we are definitely installing test dependencies :/

Reply:
nope, see https://github.com/DFE-Digital/early-careers-framework/blob/develop/.github/workflows/test.yml
If anything I'd like to remove the test dependencies, not add the dev ones

Reply:
Aside from fluff it production, it will make our builds take longer

Reply:
Ah, I think that last part convinced me, our builds do take a while.

Reply:
Done - a bit uneasy with this approach as I feel it could go out of date (it looks like I'm the only person using it and I'm not a permanent member of staff), but have added a note to the README to make it clear to anyone in the future that it could be an issue.